matryoshka_tester/data/containers.json

```
@@ -0,0 +1,121 @@
+[
+    {
+        "type": "openjdk-devel",
+        "repo": "devel/bci/images",
+        "image": "bci/openjdk-devel",
+        "tag": "16",
+        "version": "16"
+    },
+    {
+        "type": "openjdk-devel",
+        "repo": "devel/bci/images",
+        "image": "bci/openjdk-devel",
+        "tag": "15",
+        "version": "15"
+    },
+    {
+        "type": "openjdk-devel",
+        "repo": "devel/bci/images",
+        "image": "bci/openjdk-devel",
+        "tag": "14",
+        "version": "14"
+    },
+    {
+        "type": "openjdk-devel",
+        "repo": "devel/bci/images",
+        "image": "bci/openjdk-devel",
+        "tag": "11",
+        "version": "11"
```

There is no node 11 container.
```suggestion
        "tag": "12",
        "version": "12"
```

---

Argh, I just realized that I selected the openjdk image instead of nodejs…

---

Yeah and I made an error with it, because I was not careful during the rebase while doing my meetings  :D 
Anyway, that's fixed in the right place in another commit :)